bucoffea/scripts/run_quick.py

```diff
@@ -39,14 +48,28 @@ def main():
         # "zjetsnunu_ht1200to2500_mg_2018_v5":[
         # "data/zjetsnunu_ht1200to2500_mg_2018_v5.root"
         # ]
-        "SingleMuon_2018C" : [
-            "./data/SingleMuon_2018C.root"
+        #"SingleMuon_2018C" : [
+        #    "./data/SingleMuon_2018C.root"
+        #]
+        "vbfhinv2017" : [
+            "root://cmsxrootd.fnal.gov//store/user/aandreas/nanopost/21Aug19/VBF_HToInvisible_M125_13TeV_TuneCP5_powheg_pythia8/VBF_HToInvisible_M125_pow_pythia8_2017/190822_134734/0000/tree_1.root"
         ]
     }
 
     years = list(set(map(extract_year, fileset.keys())))
     assert(len(years)==1)
 
+    args = getDatasetType()
+    processorClass = args.type
```

Comment:
Please use snake case for variable names (`processor_class` rather than `processorClass`).
Sounds like a picky comment, I know, but I would like to introduce code style checks soon, so I would like us to try being consistent already.